src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -0,0 +1,235 @@
+      <arg name="groupId" type="group_id"/>
+      <arg name="sceneId" type="INT8U"/>
+      <arg name="transitionTime" type="INT16U" presentIf="status==0"/>
+      <arg name="sceneName" type="CHAR_STRING" presentIf="status==0"/>
+      <arg name="extensionFieldSets" type="SceneExtensionFieldSet" array="true" presentIf="status==0"/>
+    </command>
+    
+    <command source="server" code="0x02" name="RemoveSceneResponse" optional="false" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast RemoveScene command
+      </description>
+      <arg name="status" type="ENUM8"/>
+      <arg name="groupId" type="group_id"/>
+      <arg name="sceneId" type="INT8U"/>
+    </command>
+    
+    <command source="server" code="0x03" name="RemoveAllScenesResponse" optional="false" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast RemoveAllScenes command
+      </description>
+      <arg name="status" type="ENUM8"/>
+      <arg name="groupId" type="group_id"/>
+    </command>
+    
+    <command source="server" code="0x04" name="StoreSceneResponse" optional="false" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast StoreScene command
+      </description>
+      <arg name="status" type="ENUM8"/>
+      <arg name="groupId" type="group_id"/>
+      <arg name="sceneId" type="INT8U"/>
+    </command>
+    
+    <command source="server" code="0x06" name="GetSceneMembershipResponse" optional="false" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast GetSceneMembership command
+      </description>
+      <arg name="status" type="ENUM8"/>
+      <arg name="capacity" type="INT8U"/>
+      <arg name="groupId" type="group_id"/>
+      <arg name="sceneList" type="INT8U" array="true" optional="true" presentIf="status==0"/>
+    </command>
+
+  <command source="server" code="0x40" name="EnhancedAddSceneResponse" optional="true" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast EnhancedAddScene command
+      </description>
+      <arg name="status" type="ENUM8"/>
+      <arg name="groupId" type="group_id"/>
+      <arg name="sceneId" type="INT8U"/>
+    </command>
+
+    <command source="server" code="0x41" name="EnhancedViewSceneResponse" optional="true" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast EnhancedViewScene command
+      </description>
+      <arg name="status" type="ENUM8"/>
+      <arg name="groupId" type="group_Id"/>
+      <arg name="sceneId" type="INT8U"/>
+      <arg name="transitionTime" type="INT16U"/>
+      <arg name="sceneName" type="CHAR_STRING"/>
+      <arg name="extensionFieldSets" type="SceneExtensionFieldSet" array="true"/>
```

Comment:
These three fields need to be optional, right?

Reply:
fixed